CHANGELOG.md

```diff
@@ -79,6 +79,11 @@ the error object:
 `@cumulus/api-client/granules.deleteGranule` to control the retry
 behavior
 
+### Changed
```

Comment:
This header is a duplicate. There is already a "Changed" header above

Reply:
👍 https://github.com/nasa/cumulus/pull/2300/commits/ccf13ed4065724577e5bad76f9f1870f00f69f41